docs/source/developer/committer-notes.rst

```diff
@@ -0,0 +1,162 @@
+* For patch releases, don’t create a new release branch, use the existing branch for the corresponding major or minor version. For example, version = ``1.2.1`` → branch ``release/1.2.x``.
+* For all releases, create a tag. Tags have the naming scheme ``v<MAJ>.<MIN>.<PAT>[-rc.<NUM>]``. For example, for version = ``1.2.0``, tag = ``v1.2.0``. (Note: Tag names *do* contain the ``-rc.<NUM>`` suffix, when doing an RC release.)
+* Use RC releases to validate the system end-to-end before creating the GM release. There should be at least one successful RC release before creating the GM release.
+* Use a GitHub Release to initiate each OpenDP release. This will run the GitHub Workflows that handle the build and publish process (see below).
+
+Playbook
+--------
+
+#. Identify names:
+
+   ==============  ===============================  =================  ==============================
+   Item            Format                           Example            Notes
+   ==============  ===============================  =================  ==============================
+   base version    ``<MAJ>.<MIN>.<PAT>``            ``1.2.0``          base for RC and GM versions
+   release branch  ``release/<MAJ>.<MIN>.x``        ``release/1.2.x``  branch used for all iterations
+   RC version      ``<MAJ>.<MIN>.<PAT>-rc.<NUM>``   ``1.2.0-rc.1``     incremented for each RC
+   RC tag          ``v<MAJ>.<MIN>.<PAT>-rc.<NUM>``  ``v1.2.0-rc.1``
+   GM version      ``<MAJ>.<MIN>.<PAT>``            ``1.2.0``          same as base version
+   GM tag          ``v<MAJ>.<MIN>.<PAT>``           ``v1.2.0``
+   ==============  ===============================  =================  ==============================
+
+#. Create/update the release branch:
+
+   * Major or minor release ONLY: Create a *new* release branch, based on the desired point in ``main``.
+   * Patch release ONLY: Use the *existing* branch from the previous major or minor release, and cherry-pick changes from ``main`` into the release branch.
+
+#. Set the RC number to 1.
+#. Specify the version for this iteration: ``<MAJ>.<MIN>.<PAT>[-rc.<NUM>]``
+#. Update the version field(s) in the following files:
+
+   * ``VERSION``
+   * ``rust/opendp/Cargo.toml``
+   * ``rust/opendp-ffi/Cargo.toml`` (two entries!!!)
+   * ``python/setup.cfg`` (TODO: automate this from sphinx conf.py?)
+
+#. Commit the version number changes to the release branch.
+#. Create a GitHub Release with the following parameters:
+
+   :Tag version: ``v<MAJ>.<MIN>.<PAT>[-rc.<NUM>]``
+   :Target: ``release/<MAJ>.<MIN>.<PAT>[-rc.<NUM>]``
+   :Release title: ``OpenDP <MAJ>.<MIN>.<PAT>[-rc.<NUM>]``
+   :Describe this release: <RELASE NOTES LINK>
```

Comment:
```suggestion
   :Describe this release: <RELEASE NOTES LINK>
```

Reply:
It might be worth mentioning somewhere that we're planning on using GitHub's release notes feature (as far as I know). So the link is something like https://github.com/opendp/opendp/releases/tag/v0.1.0-rc.8 I assume.

There's no version control for GitHub release notes. Oh well.